app/src/main/java/com/example/peterchu/watplanner/Networking/ApiInterface.java

```diff
@@ -15,28 +15,21 @@
 
 public interface ApiInterface {
     @GET("terms/{termId}/courses.json")
-    Call<CourseResponse> getCourses(@Path("termId") String termId, @Query("key") String apiKey);
+    Call<CourseResponse> getCourses(
+            @Path("termId") String termId,
+            @Query("key") String apiKey);
 
-    @GET("terms/{termId}/{subject}/{catalogNumber}/schedule.json")
+    @GET("courses/{subject}/{catalogNumber}/schedule.json")
     Call<CourseScheduleResponse> getCourseSchedule(
-            @Path("termId") String termId,
             @Path("subject") String subject,
             @Path("catalogNumber") String catalogNumber,
             @Query("key") String apiKey
     );
 
-    @GET("terms/{termId}/{subject}/schedule.json")
```

Comment:
Yeah, that's the only time it's needed to be retrieved. You can view the call in `CourseDetailPresenter`